lxd/networks.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/blang/semver"
```

Comment:
Can't do that, it's a new package which hasn't gone through a security review.

We may be able to make our version checking code from LVM generic to cover this case.